lruttl/lru.py

```diff
@@ -3,15 +3,19 @@
 
 
 class LRUCache(object):
-    def __init__(self, size):
+    def __init__(self, size, default_ttl=None):
         self.size = size
+        self.default_ttl = default_ttl
         self.cache = OrderedDict()
 
     def __contains__(self, key):
         return self.get(key) is not None
 
     def __getitem__(self, key):
         return self.get(key)
+    
+    def __setitem__(self, key, value):
+        self.set(key, value, self.default_ttl)
```

Comment:
I am not sure if we want to support both `__setitem__` and `set`, since it might be good to just have one way of interacting with the object.

Also `__setitem__` doesn't support extra parameters (or at least not in a very logical manner), so it would not allow for changing the ttl per key 🤔